test/evergreen/compatibility_test_for_mongodb_releases.sh

```diff
@@ -114,15 +108,15 @@ run()
 	(build_rel 3.4) || return 1
 	(build_rel 3.6) || return 1
 	(build_rel 4.0) || return 1
+	(build_rel 4.2) || return 1
 	(build_rel develop) || return 1
-	(build_rel develop-timestamps) || return 1
 
 	# Verify forward/backward compatibility.
 	(verify 3.4 3.6) || return 1
 	(verify 3.6 4.0) || return 1
+	(verify 4.0 4.2) || return 1
 	(verify 4.0 develop) || return 1
```

Comment:
I believe we only need to support and verify compatibility between 2 adjacent releases. If `develop` here is equivalent to 4.4, this line should be removed?

Reply:
Just checked with @bvpvamsikrishna and you are correct - we should only be testing 4.4 against 4.2.